test/dlc_tests/sanity/test_remote_function_compatibility.py

```diff
@@ -0,0 +1,40 @@
+from packaging.version import Version
+from packaging.specifiers import SpecifierSet
+import pytest
+
+from invoke import run
+from test.test_utils import get_python_version_from_image_uri
+
+
+@pytest.mark.usefixtures("sagemaker_only")
+@pytest.mark.model("N/A")
+@pytest.mark.integration("remote_function")
+def test_remote_function(image):
+    """
+    Test to check compatibility of sagemaker training images with sagemaker remote function.
+    """
+    python_version = get_python_version_from_image_uri(image).replace("py", "")
+    python_version = int(python_version)
+
+    if python_version < 37 or "training" not in image:
+        pytest.skip(
+            f"Skipping remote function compatibility test for {image}. Test only for training images with Python>3.6"
+        )
+
+    repo_name, image_tag = image.split("/")[-1].split(":")
```

Comment:
Please use the pre-existing helper function: https://github.com/aws/deep-learning-containers/blob/03aa3331cb81696e6d4b8d048f1223bd3b249e7f/test/test_utils/__init__.py#L1842

```suggestion
    repo_name, image_tag = get_repository_and_tag_from_image_uri(image_uri)
```

Reply:
Using `get_container_name` helper function instead, which covers this.